plugins/inputs/neptune_apex/README.md

```diff
@@ -0,0 +1,145 @@
+# neptune_apex Input Plugin
+
+The neptune_apex Input Plugin collects real-time data from the Apex's status.xml page.
+
+### Configuration
+
+```toml
+[[inputs.neptune_apex]]
+  ## The Neptune Apex plugin reads the publicly available status.xml data from a local Apex.
+  ## Measurements will be logged under "apex".
+
+  ## The hostname/IP of the local Apex(es). If you specify more than one server, they will
+  ## be differentiated by the "hostname" tag.
+  servers = [
+    "apex.local",
+  ]
+
+  ## The response_timeout specifies how long to wait for a reply from the Apex.
+  #response_timeout = "5s"
+
+```
+
+### Metrics
+
+The [Neptune Apex](https://www.neptunesystems.com/) controller family allows an aquarium hobbyist to monitor and control
+their tanks based on various probes. The data is taken directly from the /cgi-bin/status.xml at the interval specified 
+in the telegraf.conf configuration file. 
+
+No manipulation is done on any of the fields to ensure future changes to the status.xml do not introduce conversion bugs
+to this plugin. When reasonable and predictable, some tags are derived to make graphing easier and without front-end 
+programming. These tags are clearly marked in the list below and should be considered a convenience rather than authoritative.
+
+- neptune_apex (All metrics have this measurement name)
+  - tags:
+    - host (mandatory, string) is the host on which telegraf runs.
+    - hostname (mandatory, string) contains the hostname of the apex device. This can be used to differentiate between 
```

Comment:
How do you feel about calling this `source`, this is tag that we are planning to standardize on to represent the hostname of the device being monitored, see #4413.

Reply:
Sure thing!